### PR TITLE
Fix non-platform thread issue.

### PR DESCRIPTION
**Pull request by hyue7**
fix issue in video_player and video_player_videohole:
```
[E] [ERROR:flutter/shell/common/shell.cc(1004)] The 'flutter.io/videoPlayer/videoEvents2' channel sent a message from native to Flutter on a non-platform thread. Platform channel messages must be sent on the platform thread. Failure to do so may result in data  loss or crashes, and must be fixed in the plugin or application code creating that channel.
See https://docs.flutter.dev/platform-integration/platform-channels#channels-and-platform-threading for  more information. [E]
```

### CODE REVIEW

File: packages/video_player/tizen/src/video_player.cc [hunk 1]
```diff
@@ -205,6 +212,45 @@ VideoPlayer::~VideoPlayer() {
   }
 }
 
+void VideoPlayer::ExecuteSinkEvents() {
```
Comment (swift-kim) [inline packages/video_player/tizen/src/video_player.cc:215]: How about
```suggestion
void VideoPlayer::SendPendingEvents() {
```

File: packages/video_player/tizen/src/video_player.cc [hunk 2]
```diff
@@ -205,6 +212,45 @@ VideoPlayer::~VideoPlayer() {
...
+  while (!error_event_queue_.empty()) {
+    if (event_sink_) {
+      event_sink_->Error(error_event_queue_.front().first,
+                         error_event_queue_.front().second);
+    }
+    error_event_queue_.pop();
+  }
+}
+
+void VideoPlayer::PushEvent(flutter::EncodableValue encodable_value) {
```
Comment (swift-kim) [inline packages/video_player/tizen/src/video_player.cc:233]: Will `std::queue::push` involve additional copying?
```suggestion
void VideoPlayer::PushEvent(const flutter::EncodableValue& encodable_value) {
```

File: packages/video_player/tizen/src/video_player.cc [hunk 3]
```diff
@@ -205,6 +212,45 @@ VideoPlayer::~VideoPlayer() {
...
+  if (event_sink_ == nullptr) {
```
Comment (swift-kim) [inline packages/video_player/tizen/src/video_player.cc:234]: Just for consistency
```suggestion
  if (!event_sink_) {
```

**Review by swift-kim — COMMENTED** [review]
Otherwise looks good to me. I'm afraid other plugins may have similar issues.

File: packages/video_player/tizen/src/video_player.cc (hunk 1 above)
Comment (hyue7) [reply packages/video_player/tizen/src/video_player.cc]: Fixed!

File: packages/video_player/tizen/src/video_player.cc (hunk 2 above)
Comment (hyue7) [reply packages/video_player/tizen/src/video_player.cc]: Will std::queue::push involve additional copying? >>>>> I think will not.
Fixed!

File: packages/video_player/tizen/src/video_player.cc (hunk 3 above)
Comment (hyue7) [reply packages/video_player/tizen/src/video_player.cc]: Fixed!

**Review by swift-kim — APPROVED** [review]

**Review by xiaowei-guan — APPROVED** [review]

**Pull request merged**